src/locutus_util/seed_etl/seed_data_etl.py

```diff
@@ -0,0 +1,113 @@
+"""
+Runs off of the data/seed_etl/seed_config.yml
+For more information, see /locutus_util/seed_etl/README
+
+Run examples
+`python seed_etl/seed_database.py`
+Options: 
+-e change the baseurl from localhost to another url
+-a change the default action from seeding the db to deleting from the db.
+
+"""
+
+import argparse
+import csv
+import requests
+from locutus_util.helpers import read_file, delete_codes
+from locutus_util.common import SEED_ETL_DIR
+import logging
+
+# Set up logging
+logging.basicConfig(level=logging.INFO)
+logger = logging.getLogger(__name__)
+
+def format_for_loc(file_path):
+    terminology_data = {}
+
+    with open(file_path, mode="r", newline="") as csvfile:
+        reader = csv.DictReader(csvfile)
+        for row in reader:
+            terminology_id = row["terminology_id"]
+            if terminology_id not in terminology_data:
+                terminology_data[terminology_id] = {
+                    "id": row["terminology_id"],
+                    "description": row["terminology_description"],
+                    "name": row["terminology_name"],
+                    "url": row["system"],
+                    "resource_type": row["terminology_resource_type"],
+                    "codes": [],
+                }
+            terminology_data[terminology_id]["codes"].append(
+                {
+                    "code": row["code"],
+                    "display": row["display"],
+                    "description": row["description"],
+                    "system": row['system']
+                }
+            )
+    return terminology_data
+
+def save_terminology(base_url, terminology):
+    response = {}
+    for keys, values in terminology.items():
+        t_id = values.get('id')
+        endpoint = f"{base_url}/api/Terminology/{t_id}"
+        headers = {"Content-Type": "application/json"}
+        try:
+            logger.info(endpoint)
+            res = requests.put(endpoint, json=values, headers=headers)
+            response[keys] = {"status_code": res.status_code, "response": res.text}
+            if res.status_code != 200:
+                logger.error(f"Failed to save terminology {keys}: {res.status_code} - {res.text}")
+            else:
+                logger.info(f"Successfully saved terminology {keys}")
+        except Exception as e:
+            logger.error(f"Error while saving terminology {keys}: {e}")
+    return response
+
+
+def main():
+    parser = argparse.ArgumentParser(description="Load CSV data into Firestore.")
+    parser.add_argument(
+        '-e',
```

Comment:
Just a suggestion, but I've been using "-db" "--database-uri" and have assumed that the full database URI would be present (which includes the name of the database at the end, such http://localhost:8080/locutus)